CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#### 5.0.1
```

Comment:
It looks like 5.0.0 was never published, https://pub.dev/packages/process/versions.

Lets move this back to the 5.0.0 section and then release 5.0.0